pkg/products/bamboo/locals.tf

```diff
@@ -1,5 +1,6 @@
 locals {
-  product_name = "bamboo"
+  product_name       = "bamboo"
+  helm_chart_version = "0.0.2"
```

Comment:
I am thinking if we move the helm chart version as one of the configurable items. It makes the turn key solution independent of product version. We already have enough complexity in upgrade process. Mixing the infrastructure and product will make it even more complex. 
This is an idea and no action is required here. But we can open a discussion to consider the upgrade matter with terraform project.